oe_time_caching.info.yml

```diff
@@ -3,4 +3,4 @@ description: OpenEuropa Time Caching provides time-based cache tags and invalida
 package: OpenEuropa
```

Comment:
Would be good to wrap module's name and description by single quote as in https://github.com/openeuropa/oe_webtools/pull/156/files#diff-c61f63280f6689204a18fcb67a674514314c749361707af3cd04340c1af1fadc.